cmd/interactive.go

```diff
@@ -0,0 +1,245 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"runtime"
+	"strconv"
+
+	"github.com/Naman1997/discovr/internal"
+	"github.com/charmbracelet/huh"
+	"github.com/google/gopacket/pcap"
+)
+
+var (
+	// TODO: Fix var names
+	Scantype        string
+	SelectInterface string
+	Exportpath      string
+	duration        int
+	durationStr     string
+	ip              string
+	ports           string
+	osdet           bool
+	placeholder     string
+	regionselect    string
+)
+
+const awsIPRangesURL = "https://ip-ranges.amazonaws.com/ip-ranges.json"
+
+type Adapter struct {
+	transport   string
+	description string
+}
+
+type awsIPRanges struct {
+	Prefixes []struct {
+		Region string `json:"region"`
+	} `json:"prefixes"`
+}
+
+func CheckOsPathPlaceholder() string {
+	switch runtime.GOOS {
+	case "windows":
+		placeholder = "default = no result | output.csv | C:\\Output\\Nmap.csv"
+	case "linux":
+		placeholder = "default = no result | ./Nmap.csv"
+	default:
+		placeholder = "default = no result"
+	}
```

Comment:
What happens if we run this on MacOS? Perhaps we should set Linux placeholder as the default value here?